pycytominer/cyto_utils/features.py

```diff
@@ -67,14 +76,36 @@ def label_compartment(cp_features, compartment, metadata_cols):
 
 
 def infer_cp_features(
-    population_df, compartments=["Cells", "Nuclei", "Cytoplasm"], metadata=False
+    population_df,
+    compartments=["Cells", "Nuclei", "Cytoplasm"],
+    metadata=False,
+    image_features=False,
 ):
+    """Given a dataframe, output features that we expect to be Cell Painting features.
+
+    Parameters
+    ----------
+    population_df : pandas.core.frame.DataFrame
+        DataFrame from which features are to be inferred.
+    compartments : list of str, default ["Cells", "Nuclei", "Cytoplasm"]
+        Compartments from which Cell Painting features were extracted.
+    metadata : bool, default False
+        Whether or not to infer metadata features.
+    image_features : bool, default False
+        Whether or not the profiles contain image features.
+
+    Returns
+    -------
+    features: list of str
+        List of Cell Painting features.
     """
-    Given a dataframe, output features that we expect to be cell painting features
-    """
+
     compartments = convert_compartment_format_to_list(compartments)
     compartments = [x.title() for x in compartments]
 
+    if image_features:
+        compartments = ["Image"] + compartments
```

Comment:
```suggestion
        compartments = list(set(["Image"] + compartments))
```

Just to avoid a user duplicating Image in the compartments

Reply:
https://github.com/cytomining/pycytominer/pull/168/commits/078e5281086d0650cb8a689c199c3103d1998407